aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -337,7 +337,7 @@ SparseTensor dense_to_sparse(const Tensor& self, int64_t sparse_dim){
 
 // NB: Dropped the resizeNd variants
 
-Tensor sparse_to_dense(const SparseTensor& self) {
+Tensor sparse_to_dense(const SparseTensor& self, c10::optional<ScalarType> dtype) {
```

Comment:
The new flag is completely ignored for here. Is that expected?

Reply:
This is just for dispatch which should have same numbers of inputs.

Reply:
In this case please error out if dtype is provided, as we prefer not to have situations when user passing ignored argument.